Remotion/Web/Core/UI/Controls/ButtonType.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Remotion.Web.UI.Controls
+{
+  /// <summary>
+  /// Defines how a <see cref="WebButton"/> is displayed on the page.
```

Comment:
Do you want to just add `DropDownMenu` in the next commit or update the description to not include the type name? maybe just "button face"?